Twitch Discord Bridge huge refactoring (#5)
ollprogram/TwitchDiscordBridge: .github/pull_request_template.md

## 📝 Description
Briefly describe what this Pull Request does.

---

## 🎯 Purpose / Context
Why is this change necessary?  

> What does it fix, improve, or prepare for?

Reviewer: ```suggestion
What is it fixing, improving, or preparing for?
```
